src/controllers/auth.js

```diff
@@ -219,28 +219,35 @@ const obtainAccessToken = async (req, res) => {
     }
 
     // check if the password is valid
-    let user = users[0];
+    const user = users[0];
     const isMatch = await comparePasswords(password, user.hashed_password);
 
     if (!isMatch) {
       return res.status(401).send({ message: 'Invalid username or password' });
     }
 
-    // get the user roles from the DB
-    const userRoles = rowService.get({
+    // get the users role from the DB
+    const usersRole = rowService.get({
       tableName: '_users_roles',
       whereString: 'WHERE user_id=?',
       whereStringValues: [user.id],
     });
 
-    if (userRoles < 0) {
-      return res.status(404).send({ message: 'Default role not found' });
-    }
+    const roleId = usersRole[0].role_id;
```

Comment:
What if no usersRole found?

Reply:
I have pushed a new change and a validation is added here